pkg/driver/clickhouse/clickhouse_test.go

```diff
@@ -12,14 +12,19 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func testClickHouseDriver(t *testing.T) *Driver {
-	u := dbutil.MustParseURL(os.Getenv("CLICKHOUSE_TEST_URL"))
+func testClickHouseDriverURL(t *testing.T, url string) *Driver {
```

Comment:
The function `testClickHouseDriverURL` is declared here but used only in `clickhouse_cluster_test.go`. Maybe it is ok

Reply:
Yeah also noticed it, decided on creating a common utils file named: clickhouse_testutils.go
and put all the reusable functions there